README.md

```diff
@@ -29,7 +29,7 @@ npm run migrate:latest
 Scrape the Fall 2015 catalog and load it into the database:
 
 ```bash
-node lib/scripts/scrape_catalog.js 201590
+node --harmony lib/scripts/scrape_catalog.js 201590
```

Comment:
Shouldn't need this on v4.2. https://github.com/coursereviews/schedule/blob/master/package.json#L10.